### DISCUSSION TITLE
Cleaning up of environment files

### DISCUSSION THREAD

**Discussion by VeckoTheGecko** [Parcels-code/Parcels#1289] [Q&A] [answered]
I just realised that I've misunderstood how the recipe for building parcels for conda forge works, and that it is actually maintained in [conda-forge/parcels-feedstock](https://github.com/conda-forge/parcels-feedstock/blob/main/recipe/meta.yaml)) and not in this repo.

**Am I correct in saying that all the `environment_py3_*.yml` files are solely for dev/CICD purposes?**

If yes, then my initial motivation of separating out core, dev (doc, and test) dependencies into different files is invalid. Putting core and dev dependencies all in the same file is more in line with other codebases I've now seen (eg. [pandas](https://github.com/pandas-dev/pandas/blob/main/environment.yml)).

Although maintaining 3 separate env files when adding dependencies results in some duplication, I think having singular env files would make setup easier (and also be compatible with platforms like ReadTheDocs, which only support one env file).

PS. Sorry for proposing to undo a PR I made only days after it was merged. Always learning!

**Comment by erikvansebille** [comment, marked as answer]
Hi @VeckoTheGecko, good point. Yes, these environment_* files are not necessary for the condo package itself. However, we do recommend users who want to install a developer version to also use these files. See 'installation for developers' at https://oceanparcels.org/#installing. Since these developers would probably also want the dev-tools, putting them back in the same file might be a good idea. So please propose a PR that reverts #1283, indeed ;-)
